src/core/mac/mac.cpp

```diff
@@ -499,10 +505,14 @@ void Mac::TransmitDoneTask(bool aRxPending, ThreadError aError)
     {
     case kStateActiveScan:
         mAckTimer.Start(mScanDuration);
+        mCounters.mTxBeaconRequest++;
+        mCounters.mTxNoAckRequested++;
```

Comment:
Should probably move this to the top of `Mac::SentFrame` and count `mTxAckRequested` and `mTxNoAckRequested` based on checking `sendFrame.GetAckRequest()`.